src/main/java/xyz/aadev/generitech/common/config/ConfigWorldGen.java

```diff
@@ -125,123 +123,123 @@ public static void syncConfig(Configuration config) {
         final String DESC_BIOME_RESTRICTION_TYPE = "Biome Restriction Type (Either 'blacklist' or 'whitelist')";
         final String DESC_BIOME_RESTRICTION_LIST = "Biome IDs to restrict ore spawning in";
 
-            prop = config.get(worldgenCat, "enableRetroGen", true);
-            prop.setComment(DESC_RETROGEN_ENABLED);
-            boolean retrogenEnabled = prop.getBoolean();
+        prop = config.get(worldgenCat, "enableRetroGen", true);
+        prop.setComment(DESC_RETROGEN_ENABLED);
+        boolean retrogenEnabled = prop.getBoolean();
+        prop.setRequiresMcRestart(false);
+        propOrder.add(prop.getName());
+
+        WorldGen.setRetrogenEnabled(retrogenEnabled);
+
+        for (EnumOres ore : EnumOres.byType(EnumOreType.ORE)) {
+            String oreName = ore.getOreName();
+            String oreNameLower = oreName.toLowerCase();
+            String oreCategory = worldgenCat + "." + oreName;
+            OreConfig defaultConf = OreGenConfigDefaults.get(ore);
+            List<String> propOrderOre = Lists.newArrayList();
+
+            OreConfig oreConf;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "propOrderOre" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "propOrderOre". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1854)